cmd/trace-agent/run.go

```diff
@@ -196,6 +196,13 @@ func Run(ctx context.Context) {
 		})
 	}
 
+	api.AttachEndpoint(api.Endpoint{
+		Pattern: "/config/set",
+		Handler: func(r *api.HTTPReceiver) http.Handler {
+			return cmdconfig.UpdateConfigHandler()
```

Comment:
```suggestion
			return cmdconfig.ConfigSetHandler()
```
I'm thinking this is more intuitive since it matches the `/config/set` path. What do you think?

Reply:
It is in config package, isn't it advised not to start methods with the package name? 😅

Reply:
I think this is a fair exception. The name of the method will reflect the name of the path. If it bothers you, then call it `SetHandler`. I'd be ok with that 🙂

Reply:
Alright 🙌